global.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
Let's keep `global.json` because it ensures that developers contributing to the toolkit are using our minimum version of .NET required/recommended